doc/compute-job-configuration.md

```diff
@@ -0,0 +1,393 @@
+
+	This parameter must be specified if `log_mode=file`. The file which the compute engine will use in order to write logging information
+
+- `log_level` (Required)
+
+	Possible values: `DEBUG` (default), `INFO`, `WARNING`, `ERROR`, `CRITICAL`. Defines the log level that is used by the  compute engine.
+
+- `hadoop_log_root` (Required)
+
+   Hadoop clients log level and log appender. If the user wants the hadoop components to log via SYSLOG must make sure to define an appropriate appender  in hadoop log4j.properties file. The name of this appender must be added in this parameter.
+
+	 For example if the available appenders in the log4j.properties file are SYSLOG and console the above line will be:
+
+   ```
+	 hadoop_log_root=SYSLOG,console
+	 ```
+
+#### `[reports]`
+
+In this section we declare the specific tenant used in the installation and the set of jobs available (as we described them above in the [_"Tenant and Report configuration"_](#tenant-and-report-configuration)).
+
+- `tenant` (Required)
+
+  The name of the tenant. It must be unique within an installation.
+
+- `job_set` (Required)
+
+	A comma-separated list of the report configuration. Names are case-sensitive. For the same tenant multiple report configurations can be defined. Each report configurations is defined by a set of  topologies,metric profiles,weights,etc.
+
+#### `[sampling]`
+
+- `s_period` (Required)
+
+	The sampling period time in minutes
+
+- `s_interval` (Required)
+
+	The sampling interval time in minutes
+
+> **Note**
+>
+> the number of samples used in a/r calculations is determined by the s_period/s_interval value. Default values used
+> - `s_period = 1440`
+> - `s_interval = 5`
+>
+> so number of samples = 1440/5 = 288
+
+#### `[datastore-mapping]`
+
+This section contains various optional parameters used for correctly mapping results to expected datastore collections and fields
+
+- `service_dest={db_name}.{collection_name}` (Required)
+
+  Destination for storing service a/r reports E.g: AR.sfreports
+
+- `egroup_dest={db_name}.{collection_name}` (Required)
+
+	Destination for storing endpoint grouped a/r reports. E.g: AR.sites
+
+- `sdetail_dest={db_name}.{collection_name}` (Required)
+
+	Destination for storing status detailed results |
+
+> **Note**
+>
+> Due to nature of field/value storage in the default datastore (mongodb) a specific schema is used with abbreviated field names for storage optimization. For example:
+>
+>```
+>date->dt
+>availability profile->ap
+>```
+
+The following options define a set of mappings to shorter datastore field names and is recommended not to be changed. Will be removed in further editions.
+
+- `e_map={fieldname1},{fieldname2}...,{fieldnameN}`
+
+	When storing endpoint a/r results in mongodb compute engine uses the above field map to store the results using abbreviated fields. the default value is `e_map=dt,ap,p,s,n,hs,a,r,up,u,d,m,pr,ss,cs,i,sc`, where dt->date,a->availability etc... (recommended not to be changed)
```

Comment:
although it is recommended not to change the default value of e_map, i think it is better to explain the dt,ap,p,s,n,hs,a,r,up,u,d,m,pr,ss,cs,i,sc abbreviated fields